workload/traefik.yaml

```diff
@@ -297,3 +276,32 @@ spec:
           emptyDir: {}
       nodeSelector:
        agentpool: npuser01
+---
+apiVersion: v1
+kind: List
+metadata:
+  name: traefik-ingress-service
+items:
```

Comment:
Any reason we moved to a list here instead of just deploying this service like it was before?  Just asking for my own curiosity as I don't remember seeing us do that before anywhere (like in multi-region)